app/src/androidTest/java/sdp/moneyrun/MapInstrumentedTest.java

```diff
@@ -342,5 +360,275 @@ public void closeButtonWorks(){
         }
     }
 
+    @Test
+    public void LakeLemanIsDetectedAsInappropriateTest(){
+        double lat =46.49396808615545;
+        double lon =   6.638823143919147;
+        Location location = new Location("");
+        location.setLatitude(lat);
+        location.setLongitude(lon);
+        try (ActivityScenario<MapActivity> scenario = ActivityScenario.launch(MapActivity.class)) {
+            try {
+                Thread.sleep(10000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            scenario.onActivity(a-> {
+                a.moveCameraWithoutAnimation(lat, lon, minZoomForBuilding);
+            });
+            try{
+                Thread.sleep(10000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            scenario.onActivity(a ->{
+                assert(!a.isLocationAppropriate(location));
+              //  System.out.println("At spc appropriate returns " + a.isLocationAppropriate(location));
```

Comment:
Small detail, but I think it would be best not to leave any useless comments, the same goes for the rest of the comments in this class.